snippets/cpp.snippets

```diff
@@ -129,3 +129,6 @@ snippet lld
 	[${1}](${2}){
 		${3}
 	}
+snippet uname
```

Comment:
Using a namespace is relatively short, I do not think it needs a snippet. Making one is here
See https://github.com/honza/vim-snippets/blob/master/UltiSnips/cpp.snippets#L27.

Should use the same trigger, the UltiSnips one is much superior though.